src/test/resources/application.yml

```
@@ -0,0 +1,8 @@
+spring:
+  datasource:
+    url: jdbc:h2:mem:board
+    username: sa
+    password:
+
+  jpa:
+    show-sql: true
```

```suggestion
spring:
  datasource:
    url: jdbc:h2:mem:board
    username: sa
    password:

  jpa:
    show-sql: true
 logging:
  level:
    sql: debug
    
```

spring jdbc도 사용하고 있으니 이렇게 해주면 테스트때 sql을 보기 편할 것 같아요

---

a603097b039ce3f012b786a73b43f3cd00fde6cb
피드백 적용하였습니다!